client/cypress/support/commands.js

```diff
@@ -50,3 +50,18 @@ Cypress.Commands.add('fillInputs', (elements) => {
     cy.getByTestId(testId).clear().type(value);
   });
 });
+
+Cypress.Commands.add('dragBy', { prevSubject: true }, (subject, offsetLeft, offsetTop, force = false) => {
```

Comment:
This is actually optional as I've already changed and eventually needed to create a new drag helper for `Parameters`. LMK if you prefer it as a Cypress command or a dashboard api function.